### DISCUSSION TITLE
Please add par2cmdline-turbo to sabnzbd Script

### DISCUSSION THREAD

**Discussion by bjoerrrrn** [tteck/Proxmox#2316] [General]
Heya,

par2cmdline-turbo is a faster replacement for par2. Here is an article from the sabnzbd wiki: https://sabnzbd.org/wiki/installation/par2cmdline-turbo

I have installed it in my lxc container this way:
```
cd ~
wget https://github.com/animetosho/par2cmdline-turbo/releases/download/v1.1.1/par2cmdline-turbo-v1.1.1-linux-amd64.xz
xz -dv par2cmdline-turbo-v1.1.1-linux-amd64.xz
chmod +x par2cmdline-turbo-v1.1.1-linux-amd64
mv /usr/bin/par2 /usr/bin/par2.old
mv par2cmdline-turbo-v1.1.1-linux-amd64 /usr/bin/par2
```

**Comment by edweerdt** [comment]
Much appreciated

**Comment by ghost** [comment]
This is awesome thank you so much

**Comment by bobloadmire** [comment]
is this going to be implemented?

**Comment by bobloadmire** [comment]
I used your commands on a unprivileged LXC and everything executed successfully, but sabnzb is still saying unavailable. I previously used this successfully on a privileged lxc, related? suggestions?

Edit: interestingly:
root@sabnzbd:~# par2 --version
par2cmdline-turbo version 1.1.1

**Comment by sepl2604** [comment]
thanks @bjoerrrrn, worked like a charm with your steps described.

@bobloadmire after completing the steps, I rebooted the lxc. Did you do that? After reboot, a checkmark appeared in sabnzbd, before reboot it didn't.
